build.gradle

```diff
@@ -97,6 +110,14 @@ android {
 
     applicationVariants.all { variant ->
         variant.resValue "string", "applicationId", variant.applicationId
+
+        if (variant.buildType.name == 'nightly') {
```

Comment:
not a fan of this if; is there not a nicer place?
like in the ` nightly {}`-section?

Reply:
When I put it into the nightly {} block I get an build failed error:

![image](https://user-images.githubusercontent.com/5598124/141156490-983d8870-8e3e-4532-95bb-2bccdb6ee417.png)

Took me some time to find this solution. I'm also not a fan of it.

Reply:
If someone has a better solution, it is more than welcome.

Reply:
If it works, it works :)